modules/util/source-formatter/src/main/resources/sourcechecks.xml

```diff
@@ -1779,6 +1779,10 @@
 			<category name="Upgrade" />
 			<description name="Run code to remove 'fieldset-group' tag" />
 		</check>
+		<check name="UpgradePortletDisplayCheck">
+			<category name="Upgrade" />
+			<description name="Replace the portletDisplay by ConfigurationProviderUtil class for call getPortletInstanceConfiguration method and include themeDisplay parameter" />
```

Comment:
@TamyrisTorres, how about:

`Replaces 'PortletDisplay.getPortletInstanceConfiguration' with 'ConfigurationProviderUtil.getPortletInstanceConfiguration'`

I don't think we need to mention here the inclusion of the `themeDisplay` since this just a detail about the SF check implementation, does it make sense to you?

Reply:
resolved